src/libhook/call_result.hpp

```diff
@@ -118,11 +118,13 @@ struct CallResult
 
     virtual ~CallResult() = default;
 
-    void setResultCallParams(const drakvuf_trap_info_t* info)
+    void setResultCallParams(const drakvuf_trap_info_t* info, void* context_ = nullptr)
     {
         target_pid = info->attached_proc_data.pid;
         target_tid = info->attached_proc_data.tid;
         target_rsp = info->regs->rsp;
+        context = context_;
+
```

Comment:
Please dont do this. Instead create a new structure inheriting from PluginResult (https://github.com/tklengyel/drakvuf/blob/master/src/plugins/hook_helpers.h#L128-L136) or store the data on plugin (`this`) object.

Reply:
Fixed